waku/protocol/v2/waku_filter.nim

```diff
@@ -109,7 +112,7 @@ method init*(wf: WakuFilter) =
 
     let value = res.value
     if value.push != MessagePush():
```

Comment:
What are we trying to compare here? And why the `not`?
I'm guessing you want the type of RPC message, but this will just instantiate the default values (empty message basically).

Reply:
no, we are trying to see if push is empty or not.